### PR TITLE
Login: UI fixes based on design review

### LINKED ISSUE #7547: Design adjustments for login prologue
Comparison of current implementation of prologue to design:

screen shot 2017-07-13 at 1 14 52 pm

Notes:

- Let's make the left/right margin on the text 10% of the screen width -- this translates to ~41px on the Plus-sized screens shown here. This should reduce to 32px on the SE-sized screens, I think.
- Text should be `Title 3` in size; medium weight.
- Reduce the animation height slightly, to add more whitespace above/below the text.
- Add a shadow to the background behind the buttons.
- Can we set the status bar color to white? Or is black/white chosen automatically by the system based on background color?
- Set the animations to only play once each time they appear.

I've updated the Stats and Notifications animations so that they look right when they don't repeat.

Stats: https://cldup.com/PltMKRa9hz.json
Notifications: https://cldup.com/OE-Exjq1Gm.json

### PR DESCRIPTION
**Pull request by nheagy**
Fixes #7547

Several small fixes so that the UI more closely matches the mockups.

iPhone SE | iPhone Plus
-------|--------
simulator screen shot jul 13 2017 10 00 30 pm|simulator screen shot jul 13 2017 9 59 39 pm

simulator screen shot jul 13 2017 10 07 43 pm

To test:
- Compare the prologue screen to feedback in #7547
- Compare other login screens to feedback

### Note:
The font for bottom text buttons on magic link screens are fixed in #7549

Needs review: @aerych

### CODE REVIEW

**Comment by folletto** [comment]
I did a test run, looks good. :) 

I just noticed a small regression in the 1Password icon, which is now too close to the edge:

screen shot 2017-07-14 at 13 14 15



Should have a right spacing equal to the left spacing of the icons on the left.

screen shot 2017-07-14 at 13 15 23

File: WordPress/Classes/ViewRelated/NUX/LoginProloguePromoViewController.swift [hunk 1]
```diff
@@ -21,6 +25,11 @@ class LoginProloguePromoViewController: UIViewController {
                 return
         }
 
+        stackView?.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 0.8).isActive = true
+        stackView?.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        stackView?.heightAnchor.constraint(equalTo: view.heightAnchor, multiplier: 0.87).isActive = true
+        stackView?.spacing = 36.0
```
Comment (aerych) [inline WordPress/Classes/ViewRelated/NUX/LoginProloguePromoViewController.swift]: Maybe constants for these values?

File: WordPress/Classes/ViewRelated/NUX/LoginViewController.swift [hunk 2]
```diff
@@ -11,24 +13,76 @@ class LoginViewController: NUXAbstractViewController {
...
         navigationItem.titleView = imageView
     }
 
+    /// Configures instruction label font
+    ///
+    func styleInstructions() {
+        // iOS won't return UIFontWeightMedium for dynamic system font :(
+        // -So instead get the dynamic font size, then ask for the non-dynamic font at that size
+        let fontToGetSize = WPStyleGuide.fontForTextStyle(.subheadline)
+        instructionLabel?.font = UIFont.systemFont(ofSize: fontToGetSize.pointSize, weight: UIFontWeightMedium)
```
Comment (aerych) [inline WordPress/Classes/ViewRelated/NUX/LoginViewController.swift]: This same trick is done in the prologue vc too.  Maybe factor into a WPStyleGuide extension for reuse?

File: WordPress/Classes/ViewRelated/NUX/LoginViewController.swift [hunk 3]
```diff
@@ -11,24 +13,76 @@ class LoginViewController: NUXAbstractViewController {
...
+    /// Sets up the help button and the helpshift conversation badge.
+    ///
+    override func setupHelpButtonAndBadge() {
+        NotificationCenter.default.addObserver(self, selector: #selector(NUXAbstractViewController.handleHelpshiftUnreadCountUpdated(_:)), name: NSNotification.Name.HelpshiftUnreadCountUpdated, object: nil)
+
+        let customView = UIView(frame: helpButtonContainerFrame)
+
+        helpButton = UIButton(type: .custom)
+        //let helpButton = UIBarButtonItem(title: NSLocalizedString("Help", comment: "Help button"), style: .plain, target: self, action: #selector(NUXAbstractViewController.handleHelpButtonTapped(_:)))
+        //helpButton.setImage(UIImage(named: "btn-help"), for: UIControlState())
```
Comment (aerych) [inline WordPress/Classes/ViewRelated/NUX/LoginViewController.swift]: Do we need these commented lines for anything?  Maybe nuke'em?

File: WordPress/Classes/ViewRelated/NUX/LoginProloguePromoViewController.swift (hunk 1 above)
Comment (nheagy) [inline WordPress/Classes/ViewRelated/NUX/LoginProloguePromoViewController.swift]: Ah, I meant to and totally forgot. Thanks!

**Comment by aerych** [comment]
Howdy @nheagy!  I left a few nitpicky comments (which you've already seen).  I just took things for a spin in the iPhone and iPad simulators and I'm not seeing anything amiss other than what @folletto pointed out. 
Looks good!

**Comment by nheagy** [comment]
@folletto Nice catch. Also fixed up the eye icon on the wpcom password screen:

1pass-icon-alignment

**Comment by folletto** [comment]
Design wise we're cool then! 👍

**Comment by nheagy** [comment]
Ok @aerych ready for a second round.

Fwiw: I fixed the eye icon positioning code in `WPWalkthroughTextField`. This will also affect the old signup UI, but since the change is purely visual and minor, I didn't fork the file.

**Comment by aerych** [comment]
> This will also affect the old signup UI, but since the change is purely visual and minor, I didn't fork the file

No worries about this from me.  In the unlikely event we need to switch back to the old flow we can remember to revert that change also.

**Comment by nheagy** [comment]
Thanks @folletto!

**Comment by aerych** [comment]
:shipit: :shipit: :shipit: :shipit: :shipit: :shipit: :shipit: :shipit: :shipit: !

**Comment by nheagy** [comment]
Thanks @aerych!

🚋 🚋 🚋 🚋 🚋 🚋 🚋 🚋 🚋

**Pull request merged**